access/src/main/webapp/WEB-INF/web.xml

```diff
@@ -113,12 +113,12 @@
 		<url-pattern>/*</url-pattern>
 	</filter-mapping>
 	
-	<filter>
+	<!--  <filter>
```

Comment:
This could should actually still be usable, might just be missing initializing some beans?

Reply:
There currently  is no bean for it to refer to ` No bean named 'storeAccessLevelFilter' is defined`

Reply:
You commented it out in the context file in this PR :)  but i understand that was because it was missing dependencies.  I've pushed a commit that reenables its and adds access control dependencies (stolen from admin webapp)